itests/org.openhab.core.config.discovery.usbserial.tests/itest.bndrun

```diff
@@ -50,4 +44,11 @@ Fragment-Host: org.openhab.core.config.discovery.usbserial
 	tec.uom.se;version='[1.0.10,1.0.11)',\
 	net.bytebuddy.byte-buddy;version='[1.9.10,1.9.11)',\
 	net.bytebuddy.byte-buddy-agent;version='[1.9.10,1.9.11)',\
-	org.mockito.mockito-core;version='[3.1.0,3.1.1)'
+	org.mockito.mockito-core;version='[3.1.0,3.1.1)',\
+	org.eclipse.jetty.http;version='[9.4.20,9.4.21)',\
+	org.eclipse.jetty.io;version='[9.4.20,9.4.21)',\
+	org.eclipse.jetty.security;version='[9.4.20,9.4.21)',\
+	org.eclipse.jetty.server;version='[9.4.20,9.4.21)',\
+	org.eclipse.jetty.servlet;version='[9.4.20,9.4.21)',\
+	org.eclipse.jetty.util;version='[9.4.20,9.4.21)',\
+	org.openhab.core.config.discovery.usbserial.linuxsysfs;version='[2.5.0,2.5.1)'
```

Comment:
The build fails because this runbundle needs to be manually removed, see the comment above:

```
# We would like to use a mock instead of the "PollingUsbSerialScanner" in tests
# So make sure 'org.openhab.core.config.discovery.usbserial.linuxsysfs' is not in runbundles
```
bnd will fail with another error when it's blacklisted. :-( Maybe you know of a better way?

Reply:
> Maybe you know of a better way?

Yes, I have one. Will create a PR after this one is included (to avoid merge conflicts).